notebooks/page view functional network/page_view_approach_economic_recovery.ipynb

```diff
@@ -0,0 +1,1507 @@
+    "Counter(x)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# frequency table for `seed0.2`\n",
+    "x = sorted([y for x, y in apprentice.items()], reverse=True)\n",
+    "Counter(x)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# extract list of pages that have a shortest page path length of >= 3\n",
+    "finding_job_filtered=[node for node, length in finding_job.items() if length >= 3]\n",
+    "apprentice_filtered=[node for node, length in apprentice.items() if length >= 3]\n",
+    "\n",
+    "# extract list of page where they have a shortest page path length on >= 3 from both `seed0` pages\n",
+    "remove=[node for node in finding_job_filtered if node not in apprentice_filtered]\n",
+    "\n",
+    "# remove list of pages from g\n",
+    "g.remove_nodes_from(remove)\n",
+    "\n",
+    "# number of nodes and esges\n",
+    "nx.info(g)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# how many of the economic recovery pages in the functional graph?\n",
+    "getNoOfTruePages(g)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# which pages are in the economic recovery list but not in the subgraph list? \n",
+    "getPagesNotInSubGraph(g)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Exploration: remove pages with the shortest page paths summed (`seed0.1` + `seed0.2`)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Could also look at the sum of the shortest page paths. However, the caveat here is that if the shortest page path length is `1` from `seed0.1` to `page A`, but `6` from `seed0.2` to `page A`, then the sum would be `7`. However, `page A` may be very relevant to the WUJ, as defined by it's shortest page path length from `seed0.1`.  \n",
+    "\n",
+    "Therefore, this method has not been chosen. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# sum all pages shortest page paths of both `seed0` nodes to get an overall shortest page path proxy metric\n",
+    "combined_paths = []\n",
+    "\n",
+    "for page, closeness in finding_job.items():\n",
+    "    for page1, closeness1 in apprentice.items():\n",
+    "        if page == page1: \n",
+    "            combined_score = closeness + closeness1\n",
+    "            combined_paths.append(page)\n",
+    "            combined_paths.append(combined_score)\n",
+    "            \n",
+    "combined_paths=dict(zip(combined_paths[::2], combined_paths[1::2]))\n",
+    "combined_paths={k: v for k, v in sorted(combined_paths.items(), key=lambda item: item[1], reverse=True)}"
```

Comment:
I wonder if there is a cleaner way to write this. It looks like you have two dicts: finding_jobs and apprentice. You're creating a new dict, in which you're finding the keys they have in common, then summing the values in each dict, where the keys match. Here is some code that does this in one line.

`set(a) & set(b)` find all the keys in common. Then it's just a case of indexing into the two dicts at the location of those keys in common and adding the values.

```
a = {'a': 3, 'd': 9, 'h': 1}
b = {'a': 1, 'h': 7, 'p': 5}

{k: a[k] + b[k] for k in set(a) & set(b)}
```
Output: `{'a': 4, 'h': 8}`